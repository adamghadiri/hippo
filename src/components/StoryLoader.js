import React from "react";
import ContentLoader from "react-content-loader";

export default () => (
  <div className="list-item">
    <ContentLoader
      height="33"
      speed={2}
      primaryColor="#f3f3f3"
      secondaryColor="#e2e2e2"
    >
      <rect x="0" y="25%" rx="1" ry="1" width="60%" height="4" />
      <rect x="0" y="75%" rx="1" ry="1" width="40%" height="3" />
    </ContentLoader>
  </div>
);
