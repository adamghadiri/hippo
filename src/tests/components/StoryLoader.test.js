import React from "react";
import { shallow } from "enzyme";
import StoryLoader from "../../components/StoryLoader";

test("should render story loader correctly", () => {
  const wrapper = shallow(<StoryLoader />);
  expect(wrapper).toMatchSnapshot();
});
