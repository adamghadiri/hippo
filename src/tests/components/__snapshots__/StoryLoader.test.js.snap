// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render story loader correctly 1`] = `<_class />`;
